tests/models/layoutlmv3/test_tokenization_layoutlmv3.py

```diff
@@ -35,7 +35,8 @@ class LayoutLMv3TokenizationTest(TokenizerTesterMixin, unittest.TestCase):
     tokenizer_class = LayoutLMv3Tokenizer
     rust_tokenizer_class = LayoutLMv3TokenizerFast
     test_rust_tokenizer = True
-    space_between_special_tokens = True
+    # determined by the tokenization algortihm and the way it's decoded by the fast tokenizers
+    space_between_special_tokens = False
```

Comment:
I made this change to solve the `test_encode_decode_with_spaces` test. To give more details it seems to me that this argument should be set according to the fast tokenizer count for a particular tokenization algorithm.